### ISSUE TITLE
Lab2

### ISSUE THREAD

**Issue by hhhhanz** [nus-cs2030/2021-s1#39]
Hi, I was trying to perform this:
![image](https://user-images.githubusercontent.com/70444061/92323447-56ffe800-f06b-11ea-8082-702e7487ae17.png)

But i get this error. What should I do?
![image](https://user-images.githubusercontent.com/70444061/92323452-6121e680-f06b-11ea-8c9c-ce4b88d9f07e.png)

**Comment by HolmesJJ** [comment]
Hi, @hhhhanz, I think you dont have the test1.jsh file?

**Comment by hhhhanz** [comment]
Oh where do I get this? I can't find it it luminus

**Comment by HolmesJJ** [comment]
Hi, @hhhhanz, you can pull all the files from school plab server.

Here is the comand 
```
scp -r [Userid]@[Hostname]:* .
```

**Comment by hhhhanz** [comment]
![image](https://user-images.githubusercontent.com/70444061/92323863-9f6cd500-f06e-11ea-94ec-cbb60c1a8f58.png)

Doesn't seem to work :(

**Comment by domlimm** [comment]
Here you go!

[Lab02 test files.zip](https://github.com/nus-cs2030/2021-s1/files/5179454/Lab02.test.files.zip)

**Comment by HolmesJJ** [comment]
> ![image](https://user-images.githubusercontent.com/70444061/92323863-9f6cd500-f06e-11ea-94ec-cbb60c1a8f58.png)
> 
> Doesn't seem to work :(

The comand can work for me. But I think have to connect to school wifi to pull.

**Comment by hhhhanz** [comment]
Yeah I thought so! :) Its okay, thank you @HolmesJJ & @domsterthebot :)

**Comment by domlimm** [comment]
> > ![image](https://user-images.githubusercontent.com/70444061/92323863-9f6cd500-f06e-11ea-94ec-cbb60c1a8f58.png)
> > Doesn't seem to work :(
> 
> The comand can work for me. But I think have to connect to school wifi to pull.

You can use the school's vpn PulseSecure or something, forgot the name

**Comment by lkldev** [comment]
> ![image](https://user-images.githubusercontent.com/70444061/92323863-9f6cd500-f06e-11ea-94ec-cbb60c1a8f58.png)
> 
> Doesn't seem to work :(

you can connect to sunfire to connect to the pe machine too. However you will need to pull twice, once from pe machine to sunfire, the other from sunfire to your own machine.
